endpoints.json

```diff
@@ -64,5 +64,19 @@
         "body": "Crumpets with butter are my favourite!",
         "article_id": 1
     }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "serves an object of a specified article with a changed vote count",
+    "queries": [],
+    "exampleResponse" : {
+      "article_id": 1,
+        "title": "Living in the shadow of a great man",
+        "topic": "mitch",
+        "author": "butter_bridge",
+        "body": "I find this existence challenging",
+        "created_at": "2020-07-09T20:11:00.000Z",
+        "votes": 58,
+        "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+    }
```

Comment:
Nice, this is everything we expect from this endpoint so all of these properties should be checked to ensure they are all coming back as expected.